### PR TITLE
Feature/improve toc scroll

### PR DESCRIPTION
**Pull request by lagartoverde**
I don't really like the solution to be honest because it involves a magic number and it has to be individually configured for every environment the editor runs on. Furthermore it has to change if the height of the topbar changes, but at least is better than before, and we can always run the previous behaviour if not configured.
As this was timeboxed I didn't want to really go further but this can be improved by maybe calculating this magic number in any way, but I don't know how right now

### CODE REVIEW

**Comment by lagartoverde** [comment]
I tested on GN and for example the magic number has to change to 200 but otherwise it works perfectly fine

**Comment by lagartoverde** [comment]
Solves https://binnenland.atlassian.net/browse/GN-4200

**Review by abeforgit — CHANGES_REQUESTED** [review]
I have to click twice on the ToC to scroll to the section now. That's a regression

File: addon/components/table-of-contents-plugin/ember-nodes/table-of-contents.ts [hunk 1]
```diff
@@ -83,7 +83,24 @@ export default class TableOfContentsComponent extends Component<EmberNodeArgs> {
       const selection = Selection.near(resolvedPos, 1);
       if (selection) {
         tr.setSelection(selection);
-        tr.scrollIntoView();
+        const coords = this.controller.mainEditorView.coordsAtPos(
+          selection.from
+        );
+        const config = this.config[0];
+        if (config.scrollContainer) {
+          const sayContainer = document.getElementsByClassName(
```
[the remark this answers is not in the document]
Comment (lagartoverde) [reply addon/components/table-of-contents-plugin/ember-nodes/table-of-contents.ts]: Yes, that was something I forgot to update to be honest, done now 👍

**Comment by lagartoverde** [comment]
> I have to click twice on the ToC to scroll to the section now. That's a regression


Seems like is not a regression as I can reproduce on master, likely something weird happening with the prosemirror selections will have a look

**Pull request merged**